cmd/util/cmd/read-badger/cmd/chunk_data_pack.go

```diff
@@ -0,0 +1,39 @@
+package cmd
+
+import (
+	"github.com/rs/zerolog/log"
+	"github.com/spf13/cobra"
+
+	"github.com/onflow/flow-go/cmd/util/cmd/common"
+	"github.com/onflow/flow-go/model/flow"
+)
+
+var flagChunkID string
+
+func init() {
+	rootCmd.AddCommand(chunkDataPackCmd)
+
+	chunkDataPackCmd.Flags().StringVarP(&flagChunkID, "chunk-id", "c", "", "the id of the chunk")
+	_ = chunkDataPackCmd.MarkFlagRequired("chunk-id")
+}
+
+var chunkDataPackCmd = &cobra.Command{
+	Use:   "chunkdata",
+	Short: "get chunk data pack by chunk ID",
+	Run: func(cmd *cobra.Command, args []string) {
+		storages := InitStorages()
+
+		chunkID, err := flow.HexStringToIdentifier(flagChunkID)
+		if err != nil {
+			log.Fatal().Err(err).Msg("malformed chunk id")
+		}
+
+		log.Info().Msgf("getting chunk data pack by chunk id: %v", chunkID)
```

Comment:
Better to print the `flagChunkID` first, so that if the chunk id is not a hex string, we could see from the log what was the flag.

The general structure could be 
```
if flagXXX != "" {
  log(flagXXX is %v)
  xxxID = parseHex(flagXXX)
  log(xxxID is %v)
  ...
  return
} 

if flagYYY != "" {
  log(flagYYY is %v)
  ...
  return
}

fatal(missing XXX or YYY)